source/includes/fundamentals/code-snippets/Filters.java

```diff
@@ -123,18 +133,21 @@ private void bitsComparison() {
 
     private void geoWithinComparison() {
         // begin geoWithinComparison
+        // Creates a Polygon to filter the results
```

Comment:
Suggestion:

I think this code comment should be omitted since it describes the code rather than the functionality.